src/litserve/api.py

```diff
@@ -143,3 +143,7 @@ def has_active_requests(self) -> bool:
 
     def has_capacity(self) -> bool:
         raise NotImplementedError("has_capacity is not implemented")
+
+    def health(self) -> bool:
+        # TODO: Discuss what this should return
```

Comment:
This needs proper documentation :)

Reply:
you mean the docstring, right?  

btw, @aniketmaurya, I also wanted to check if we keep it as `NotImplemented` or a default value with `True` is fine?

Reply:
i think default `True` would be fine for this.